### PR TITLE
[spike] DCOS-48015: [1.13] Adding Package Repository with priority 0, does not push all packages 1 priority down

### PR DESCRIPTION
**Pull request by GeorgiSTodorov**
Fix repository priority when adding a new repository.

Closes https://jira.mesosphere.com/browse/DCOS-48015

Thanks to @nLight for helping me realize that priority is called index in the API.

Once this is approved, I will create the backports.

## Testing
1. Go to Settings > Package Repositories
2. Add a new Package Repository with priority 0, for example using the name "Universe2" and the uri "https://downloads.mesosphere.com/universe/repo/repo-up-to-1.13.json.gz"
3. Verify that the new package was added at the top

## Trade-offs
None.

## Dependencies
None.

### Before
peek 2019-02-01 13-54

### After
peek 2019-02-01 13-55

### CODE REVIEW

File: plugins/catalog/src/js/repositories/RepositoriesAdd.js [hunk 1]
```diff
@@ -32,8 +32,10 @@ const getErrorMessage = (response = {}) => {
 
 const addPackageRepositoryMutation = gql`
   mutation {
-    addPackageRepository(name: $name, uri: $uri, priority: $index) {
+    addPackageRepository(name: $name, uri: $uri, index: $index) {
       name
+      uri
```
Comment (nLight) [inline plugins/catalog/src/js/repositories/RepositoriesAdd.js]: so we haven't been passing `uri` as well?

Comment (GeorgiSTodorov) [reply plugins/catalog/src/js/repositories/RepositoriesAdd.js]: Apparently not.

**Comment by nLight** [comment]
To test the PR Stub Universe can be used as a repo link:
https://universe-converter.mesosphere.com/transform?url=https://dcos-ui-universe.s3.amazonaws.com/oss/dcos-ui/latest/stub-universe-dcos-ui.json

**Comment by nLight** [comment]
We gonna need 1.12 backport :)

**Comment by GeorgiSTodorov** [comment]
I can do the backport now or during the next spike day.

File: plugins/catalog/src/js/repositories/RepositoriesAdd.js (hunk 1 above)
Comment (TattdCodeMonkey) [reply plugins/catalog/src/js/repositories/RepositoriesAdd.js]: We were passing the `uri`  that is done with the context parameter `$uri` in the mutation call. Adding uri & index on lines 37 & 38 ensures those fields are returned on the result.

**Pull request merged**